images/macos/provision/core/openjdk.sh

```diff
@@ -1,28 +1,37 @@
 #!/bin/bash -e -o pipefail
 source ~/utils/utils.sh
 
-createEnvironmentVariable() {
-    local VARIABLE_NAME=$1
-    local JAVA_VERSION=$2
-    if [[ $JAVA_VERSION == "8" ]]; then
-        JAVA_VERSION="1.${JAVA_VERSION}"
+JAVA_TOOLCACHE_PATH=$AGENT_TOOLSDIRECTORY/Java_Adoptium_jdk
+
+installJavaFromAdoptOpenJDK() {
```

Comment:
Why you remove function `createEnvironmentVariable`? I think it was more readable to have this code in the separate function

Reply:
Returned this function